pokemon-showdown

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env node
 'use strict';
 
+const spawnSync = require('child_process').spawnSync;
```

Comment:
>I'm fine with explicitly importing modules, but not individual functions from modules.

I understand where you're coming from, since it's inconsistent with how we normally import modules. I deviated from that here since we won't need any of the other functions from the module as far as I can see, unlike with modules like `fs` or `path`. I'll change this regardless

>Anyway, you still use require('child_process').spawnSync earlier, so at least be consistent. >:(

Whoops